docs/gpu_performance_tips.md

```diff
@@ -0,0 +1,84 @@
+# GPU peformance tips
+
+## Matmul precision
+
+TODO
+
+## XLA performance flags
+
+As of `jaxlib==0.4.18` (released [Oct 6
+2023](https://pypi.org/project/jaxlib/#history)), setting these XLA flags can
+improve performance. Some are related to communication between GPUs, and so are
+only relevant when running computations on multiple devices, while others are
+related to code generation on each device.
+
+Some of these may be set by default in future releases.
```

Comment:
Worth mentioning maybe that these aren't stable and the names can change between releases (or a flag being deleted)?

Reply:
Great idea!

Reply:
Done!